bin/codemods/README.md

```diff
@@ -0,0 +1,93 @@
+# Welcome to Calypso's Codemod Collection!
+
+## What are codemods?
+
+Code modification scripts, also known as codemods, are transformation scripts that can simultaneously modify multiple files with precision and reliability. Codemods were popularized by [Facebook's engineering team](https://medium.com/@cpojer/effective-javascript-codemods-5a6686bb46fb) and depends greatly on Facebook's [jscodeshift](https://github.com/facebook/jscodeshift) library, which wraps over a library named [recast](https://github.com/benjamn/recast) (author of which is associated with the [Meteor](https://www.meteor.com/) project).
+
+## How to run our codemods
+
+It's easy! Our codemod script uses the following CLI:
+
+```bash
+./bin/codemods/run.js [transformation name] [target(s)]
+```
+
+```bash
+# Same as above, but using npm scripts
+npm run codemod [transformation name] [target(s)]
+```
+
+For example, if I wanted to run the `commonjs-exports` transformation on `client/devdocs/`, I can do the following:
+
+```bash
+./bin/codemods/run.js commonjs-exports client/devdocs/
+```
+
+Do you want to target files individually? We can do that, too!
```

Comment:
Why not run a few codemods at once? It seems like a legit use case and seems like a very easy addition after all those amazing refactorings done so far. It's of course not mandatory, just leaving you with another idea 😁

Reply:
For instance commonjs-exports,some-other-transform,yet-another-name

Reply:
Nice idea!

I implemented this in 011ef0859ca09205426877d680e603d93c5013fb

As a bonus we get terminal colors from the codemods by using spawnSync `options.stdio` 🎉